src/js/components/Connect/OrganizationsFollowedOnTwitter.jsx

```diff
@@ -135,7 +135,14 @@ export default class OrganizationsFollowedOnTwitter extends Component {
             <span className="position-rating__source with-popover">
               <Link to={voterGuideLink}>
                 <OrganizationTinyDisplay
-                  {...oneOrganization}
+                  showOppose={oneOrganization.showOppose}
```

Comment:
Hi @QueMona98 I like that these props are all camelCase, but you will need to go into src/js/components/VoterGuide/OrganizationTinyDisplay.jsx and update the incoming props to camelCase for some variables, like `voter_guide_image_url_tiny` and `organization_photo_url_tiny`, throughout the entire OrganizationTinyDisplay file.